libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/text/AnnotatedStrings.kt

```diff
@@ -82,3 +82,22 @@ fun buildAnnotatedStringWithStyledPart(
         end = startIndex + coloredPart.length,
     )
 }
+
+/**
+ * Convert a string to an [AnnotatedString] with colored end period if present.
+ */
+fun withColoredPeriod(
```

Comment:
If this is used only in `SunsetPage` I'd move it there instead, since it seems quite specific. WDYT?

Reply:
Yes, maybe, but I prefer to keep it here, and extend it if necessary for future use. For instance, pass a list of char that should be colored (`?`,  `!`, etc.) and pass the color as parameter.